python/setup.py

```diff
@@ -224,7 +224,7 @@ def _run_cmake(self):
             static_lib_option = ''
 
             cmake_options = [
-                '-DPYTHON_EXECUTABLE=%s' % sys.executable,
```

Comment:
Ah, could you keep this?
If we can both `PYTHON_EXECUTABLE` and `Python3_EXECUTABLE`, it works on newer and older CMake.

Reply:
For sure!